tests/profilefields/manager_test.php

```diff
@@ -0,0 +1,178 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+class manager_test extends phpbb_database_test_case
+{
+	/** @var \phpbb\config\db_text */
+	protected $config_text;
+
+	/** @var \phpbb\db\driver\driver_interface */
+	protected $db;
+
+	/** @var \phpbb\db\tools\tools */
+	protected $db_tools;
+
+	/** @var \phpbb\log\log_interface */
+	protected $log;
+
+	/** @var \phpbb\profilefields\manager */
+	protected $manager;
+
+	/** @var string Table prefix */
+	protected $table_prefix;
+
+	/**
+	 * {@inheritdoc}
+	 */
+	public function getDataSet()
+	{
+		return $this->createXMLDataSet(dirname(__FILE__).'/fixtures/manager.xml');
+	}
+
+	public function setUp()
```

Comment:
Looks like it's broke tests on 3.3/master, should be `public function setUp(): void` probably.